R/openapi-spec.R

```diff
@@ -17,8 +17,18 @@ endpointSpecification <- function(routerEndpointEntry, path = routerEndpointEntr
   funcParams <- routerEndpointEntry$getFuncParams()
   # Get the plumber decoration defined endpoint params
   endpointParams <- routerEndpointEntry$getEndpointParams()
+
+  # if legacy plumber.staticSerializers is specified then ignore all req serializer
+  # based parameters
+  if (getOption("plumber.staticSerializers", default="FALSE") == "TRUE") {
+    serializerParams <- list()
+  } else {
+    # Get the plumber serializer defined endpoint params
+    serializerParams <- routerEndpointEntry$getSerializerParams()
+  }
```

Comment:
Let's let the endpoint determine it's serializer params.

This will allow for the endpoint to have full control vs using a globally set flag. 

Counter example app: API with two routes: 1 w/ dynamic serializer and 1 w/ static serializer.

```suggestion
    # Get the plumber serializer defined endpoint params
    serializerParams <- routerEndpointEntry$getSerializerParams()
```